community/neo4j/src/test/java/upgrade/FailToStartStoreMigratorIT.java

```diff
@@ -59,6 +59,7 @@ public void shouldFailToStartWithImproperlyUpgradedPropertyKeyStore() throws Exc
         // then
         catch ( RuntimeException e )
         {
+            e.printStackTrace();
```

Comment:
Eeek. where did that come from?! must have been something I needed during debugging. Thanks for spotting it!